daemon/device-plugin/types.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2024 Intel Corp. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Also remove license blob here.

Reply:
think already removed from up to date commit? let me confirm

Reply:
confirmed - not in latest commit. a squash later will clean the history